client/src/components/Register.js

```diff
@@ -226,7 +226,7 @@ class Register extends Component {
 
     // Render loading if we don't have props yet...
     if (!registerProps) {
-      return <LoadingIndicator />;
+      return <LoadingIndicator block color="rgba(0,113,196,1)" />;
```

Comment:
Is there a way we can do this without hard coding colours in? Something with CSS and class names/component modifiers instead?

Reply:
Yeah, I wasn't super sure about this implementation, but I also wasn't sure about baking the colours into the icons / CSS, since the front-end app applies a different shade of blue to the admin interface. Maybe there should just be one shade of blue? I'll check with Jared.

Reply:
Jared was happy to unify the colours, so this is now baked into the component.